src/plugins/vis_builder/server/index.ts

```diff
@@ -17,8 +17,5 @@ export function plugin(initializerContext: PluginInitializerContext) {
 export { VisBuilderPluginSetup, VisBuilderPluginStart } from './types';
 
 export const config: PluginConfigDescriptor<ConfigSchema> = {
-  exposeToBrowser: {
-    enabled: true,
-  },
```

Comment:
Why do we no longer need this?

Reply:
Yeah same question. My understanding was we can use the config schema in the `plugin.ts` class constructor, inside public folder of the plugin only when we set `exposeToBrowser` on enabled property to true. Without this settings, is it possible to disable the visbuilder during plugin registration time?

Reply:
@manasvinibs I believe this setting is only used at registration and before browser rendering. So I believe this probably was never used unless there was custom logic in the public folder to check this config